druid/examples/hello_wasm/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "hello-wasm"
+version = "0.1.0"
+license = "Apache-2.0"
+description = "Minimal wasm example"
+repository = "https://github.com/xi-editor/druid"
+edition = "2018"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+druid = { path="../.."}
+wasm-bindgen = "0.2.60"
+console_error_panic_hook = { version = "0.1.6" }
+log = "0.4.8"
+instant = { version = "0.1", features = [ "wasm-bindgen" ] }
+
+[target.'cfg(not(target_arch="wasm32"))'.dependencies]
+simple_logger = { version = "1.6.0", default-features = false }
```

Comment:
These seem to be unused?

Reply:
More specifically `simple_logger`, `instant` and `log`.